app/forms/openid_connect_authorize_form.rb

```diff
@@ -255,4 +265,9 @@ def validate_privileges
       )
     end
   end
+
+  def highest_level_aal(aal_values)
+    default = proc { Saml::Idp::Constants::AAL1_AUTHN_CONTEXT_CLASSREF }
+    AALS_BY_PRIORITY.find(default) { |aal| aal_values.include?(aal) }
```

Comment:
```suggestion
    AALS_BY_PRIORITY.find { |aal| aal_values.include?(aal) } ||
      Saml::Idp::Constants::AAL1_AUTHN_CONTEXT_CLASSREF
```

I'm not familiar with why `#find` takes a proc for the default, but this feels more intuitive.

Reply:
I agree, that does feel more natural.